Source/MQTTnet.Extensions.ManagedClient/ManagedMqttClient.cs

```diff
@@ -527,7 +558,19 @@ async Task PublishReconnectSubscriptionsAsync()
                 if (_reconnectSubscriptions.Any())
                 {
                     var subscriptions = _reconnectSubscriptions.Select(i => new MqttTopicFilter { Topic = i.Key, QualityOfServiceLevel = i.Value });
-                    await InternalClient.SubscribeAsync(subscriptions.ToArray()).ConfigureAwait(false);
+                    List<MqttTopicFilter> subs = new List<MqttTopicFilter>();
+                    foreach (var sub in subscriptions)
+                    {
+                        subs.Add(sub);
+                        //aws only allows 8 in a single message
```

Comment:
Please move this comment to a real summary comment at property level. So all users can read that without accessing the source code.